### DISCUSSION TITLE
[Feature] Pet Recognition

### DISCUSSION THREAD

**Discussion by thekidflipper** [immich-app/immich#14442] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Goal is to be able to identify pets as well. It's something Google used to do well, but I think just having the option of just "cat" would be very neat 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Like the template requests, please search for duplicates before opening an FR. #7151
